src/sentry/assistant/guides.py

```diff
@@ -150,5 +152,54 @@
                 'target': 'member_role',
             },
         ]
-    }
+    },
+    # Ideally, this would only be sent if the organization has never
+    # customized alert rules (as per FeatureAdoption)
+    'alert-rules': {
+        'id': 5,
+        'cue': _('Tips for alert rules'),
+        'required_targets': ['alert_conditions'],
+        'steps': [
+            {
+                'title': _('Reducing alert noise'),
+                'message': _('Sentry, by default, alerts on every <i>new</i> issue via email. '
+                             'If that\'s too noisy, sending the new issue alerts to '
+                             'a service like Slack help reduce inbox noise.'),
+                'target': 'alert_conditions',
+            },
+            {
+                'title': _('Defining priority alerts'),
+                'message': _('Not alerts are equal. Create rules for frequently occuring errors or specific '
+                             'tags to escalate critical issues by alerting you via email or PagerDuty.'),
+                'target': 'alert_actions',
+            },
+            # When ownership is released
+            # {
+            #     'title': _('Ownership'),
+            #     'message': _(),
+            #     'target': None
+            # },
+            {
+                'title': _('Tweaking personal email settings'),
+                'message': _('If you\'d like to customize your <i>personal</i> alert settings, '
+                             'go to <a href="/account/settings/notifications/">Account Notifications</a>. '
+                             'There, you can choose which project\'s alert or workflow notifications you\'d like to receive.'),
+                'target': None,
```

Comment:
Can you add a comment to the top of the file about how targets behave. it's getting confusing
- target: 'non-empty-string' = step is tied to an anchor target. if the anchor doesn't exist, the step will not be shown. if the anchor exists but is of type "invisible", it will not be pinged but will be scrolled to. otherwise the anchor will be pinged and scrolled to.
- target: None = step isn't tied to any particular element, and will never be removed from the guide.

if this gets too confusing I might make a pass to clean things up

Reply:
@denamwangi, I noticed you used "invisible" anchors, which might be the better solution. Thoughts?

Reply:
documented a bit better, but still need to decide what best behavior is here.